editor/components/meta-boxes/meta-boxes-area/style.scss

```diff
@@ -1,93 +1,93 @@
 
-.editor-meta-boxes-area {
+/* Include #poststuff to override the css rules found in core. */
+#poststuff .editor-meta-boxes-area {
 	position: relative;
 
-		/* Match width and positioning of the meta boxes. Override default styles. */
-	#poststuff {
+	/* Match width and positioning of the meta boxes. Override default styles. */
+	.postbox-container {
 		margin: 0 auto;
 		padding-top: 0;
-		min-width: auto;
-	}
+		width: 100%;
 
-	#post {
-		margin: 0;
-	}
+		#post {
```

Comment:
Unrelated to this PR but I believe this one is not necessary anymore, this is probably a leftover for some old PR.